### PR TITLE
Configurable parameter groups for RDS

### PR DESCRIPTION
**Pull request by kennethjor**
Story details: https://app.clubhouse.io/7factor/story/22516

### CODE REVIEW

**Comment by kennethjor** [comment]
I ran a Terraform plan on HT stage and it succeeded. I have not tried actually deploying this to stage yet to fully verify it works.

File: variables.tf [hunk 1]
```diff
@@ -66,7 +66,12 @@ variable "engine" {
   description = "Engine name, defaults to aurora."
 }
 
-variable engine_version {
+variable "engine_version" {
```
Comment (nintendo424) [inline variables.tf:69]: Are the double quotes now necessary?

Comment (kennethjor) [reply variables.tf:69]: Not necessary, I just ran `terraform format` and it added them.

**Review by nintendo424 — APPROVED** [review]

File: variables.tf (hunk 1 above)
Comment (chris-malloy) [reply variables.tf:69]: @kennethjor it shouldn't do that, what version of terraform are you on?

Comment (chris-malloy) [reply variables.tf:69]: apparently this is intended. it doesn't do it for me. weird https://github.com/hashicorp/terraform/issues/27102

**Review by chris-malloy — APPROVED** [review]

**Review by chelseagreen — APPROVED** [review]

**Pull request merged**